icons/monitor-play.svg

```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="m10 7 5 3-5 3Z" />
-  <rect width="20" height="14" x="2" y="3" rx="2" />
+  <path d="M11 13a.75.75 0 0 1-1-.5V7.41a.75.75 0 0 1 1-.359l3.557 2.117a1 1 0 0 1 0 1.714z" />
```

Comment:
Some of those corners are a bit wonky (there are some sharp bits), this one has better curvature:
```suggestion
    <path d="M10 8a1 1 0 0 1 1.559-.832l2.996 2a1 1 0 0 1 0 1.662l-2.996 2A1 1 0 0 1 10 11.998z" />
```

On the other hand, even though your version is guideline compliant, I kind of feel like this might be a bit too much rounding, at 100% it's a bit hard to even tell that it's a triangle. 🤔

The middle one has a minor pixel perfection issue to the right, but maybe it's worth the price to make it more legible and at the same time guideline compliant (2px gap).

![image](https://github.com/lucide-icons/lucide/assets/17746067/ff7dbbe8-aaf6-4d23-82eb-62a9d8d2f734)

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M10+7.75a.75.75+0+0+1+1.142-.638l3.664+2.249a.75.75+0+0+1+0+1.278l-3.664+2.25a.75.75+0+0+1-1.142-.64z%22+/%3E%3Cpath+d=%22M12+17v4%22+/%3E%3Cpath+d=%22M8+21h8%22+/%3E%3Crect+x=%222%22+y=%223%22+width=%2220%22+height=%2214%22+rx=%222%22+/%3E&name=image"><img alt="icons" width="400" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTAgNy43NWEuNzUuNzUgMCAwIDEgMS4xNDItLjYzOGwzLjY2NCAyLjI0OWEuNzUuNzUgMCAwIDEgMCAxLjI3OGwtMy42NjQgMi4yNWEuNzUuNzUgMCAwIDEtMS4xNDItLjY0eiIgLz4KICA8cGF0aCBkPSJNMTIgMTd2NCIgLz4KICA8cGF0aCBkPSJNOCAyMWg4IiAvPgogIDxyZWN0IHg9IjIiIHk9IjMiIHdpZHRoPSIyMCIgaGVpZ2h0PSIxNCIgcng9IjIiIC8+Cjwvc3ZnPg==.svg"/><br/>Open lucide studio</a>

```suggestion
  <path d="M10 7.75a.75.75 0 0 1 1.142-.638l3.664 2.249a.75.75 0 0 1 0 1.278l-3.664 2.25a.75.75 0 0 1-1.142-.64z" />
```

Reply:
I agree that the icon is already hard to see at 100%, don't know think I'm making it with tho.